packages/beacon-state-transition/src/allForks/util/epochContext.ts

```diff
@@ -132,7 +135,6 @@ export function createEpochContext(
   // Only after altair, compute the indices of the current sync committee
   const onAltairFork = currentEpoch >= config.ALTAIR_FORK_EPOCH;
 
```

Comment:
After the loop, needs to check for 0, eg:
```ts
if (!totalActiveBalance) totalActiveBalance = BigInt(1);
```

Reply:
Done